tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/CatalogUtil.java

```diff
@@ -792,6 +791,69 @@ public static AlterTableDesc setProperty(String tableName, KeyValueSet params, A
     return alterTableDesc;
   }
 
+  /**
+   * Add partition or Drop partition
```

Comment:
The comment should be improved because this method create AlterTableDesc instance rather than adding or dropping any partition.